test/functional/Java17andUp/playlist.xml

```diff
@@ -23,13 +23,6 @@
 <playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../TestConfig/playlist.xsd">
 	<test>
 		<testCaseName>Jep389Tests_testClinkerFfi_DownCall</testCaseName>
-		<disables>
-			<disable>
-				<comment>https://github.com/eclipse-openj9/openj9/issues/14483#issuecomment-1039546787</comment>
```

Comment:
Does this change to unexclude the jdk17 test rely on anything else in this PR? It seems unrelated and should be done in a separate PR.

Reply:
Agreed and moved it to https://github.com/eclipse-openj9/openj9/pull/16599 given it is related to https://github.com/eclipse-openj9/openj9/issues/14483 (the OpenJDK MH.invokeBasic() which was solved in JIT).